SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
+
+If you believe you have found a security vulnerability in this repository, please report it to us through coordinated disclosure.
```

Comment:
Is this line necessary? CVD is a backend step that the dev team should take - it doesn't impact how users report issues, because the lines immediately following tell them the appropriate means to submitting a report.

Reply:
The line
> Please do not report security vulnerabilities through GitHub issues, discussions, or pull requests.
Specifies some examples of how issues shouldn't be reported. CVD is mentioned because it tells the user not to publicly disclose the vulnerability publicly after alerting the developer team.

Reply:
Fair enough, we can leave it then. Otherwise LGTM.